a11y-meta-display-guide/2.0/draft/guidelines/index.html

```diff
@@ -2298,40 +2298,30 @@ <h3>Extended filtering set</h3>
 		<h2>Localization</h2>
 
 		<p>
-			An ebook can be purchased in any country without limited
-			availability or additional delivery costs.
+			An ebook can be purchased in any country  and be made available through a distributor or a library.
 			Readers want consistent display of accessibility
-			information, and that’s the primary role of the display
+			information that spans multiple languages, and that’s the primary role of this display
 			guide.
-			The wording proposed in this guide has been widely discussed
+			The wording of information provided to users proposed in this guide has been widely discussed
 			by a large group representing different
-			actors of the English-speaking geographies. It has been
-			improved after proof-of-concept implementations
-			and reviewed by panels of testers. </p>
+			people in the English-speaking language group. The language has been
+			improved after proof-of-concept translations
+			and reviewed by panels of testers in several countries. </p>
 
 		<p>
-			We understand that different countries or different target
+			It is understood that different countries or different target
 			audience services may want to use specific language, and we
-			have provided flexibility in the guide to accommodate this
-			while maintaining a high level of understanding and similar
-			quality to help users in a country find likewise information
-			between two bookstores or libraries in the same language
-			area.
-			Simply translating the strings is not enough; the subtle
+			have provided flexibility in the guide to accommodate this requirement. Simply translating the strings is not enough; the subtle
 			meaning of the words and phrases of accessibility concepts
-			must be localized for maximum understanding. Therefore, we
-			intend to provide a mechanism for the publishing community
-			to provide translations that localizes the strings for
-			equally effective communication in many languages.</p>
+			must be localized for maximum understanding. Therefore, a mechanism is being provided for the publishing community
+			to collaboratively deliver translations that localizes the strings for
+			equally effective communication in any language. It is hoped that the localization area will become the defacto hub for development and distribution of language packs for each language and regional variation.</p>
 
-		<p>The <a href="../localizations">Display guide localization
-				strings page</a> list translations with
-			contextualization of the localization process.</p>
 
-		<p>The following subsections proposes a localization framework
-			to help establishing linguistic areas wordings agreed
-			between actors of the concerned geographical area. </p>
+<p><a href="https://w3c.github.io/publ-a11y/a11y-meta-display-guide/2.0/localization/">Localization development and contributions area</a></p>
```

Comment:
Shall we point the new repo? https://w3c.github.io/publ-a11y-display-guide-localizations/